### DISCUSSION TITLE
Support for Spotify Categories (genres)

### DISCUSSION THREAD

**Discussion by jazzyisj** [music-assistant/support#535] [Feature requests and ideas]
The ability to select a Spotify category (genre) to play with MASS would be a very welcome addition.

https://developer.spotify.com/documentation/web-api/reference/#/operations/get-categories

**Comment by timvdsm** [comment]
This would be absolutely great!
And even some more categories like Editor’s picks, Top tracks, Podcasts, Whats news, Liked Songs, …
In LMS it’s available if you use The Spotty integration:

![0D86D037-A9C4-4FF8-9A63-15F80709E8E6](https://user-images.githubusercontent.com/14946020/183284933-6d1719ee-18a8-460f-9fa0-0ee37dc816eb.jpeg)

**Comment by rvandet** [comment]
I would like to filter on genre because i like classical music and the rest of the family does not.
All albums are now on one big pile, so no one can find its need. Forces us to directory listings.

**Comment by OzGav** [comment]
Related to https://github.com/orgs/music-assistant/discussions/562

**Comment by OzGav** [comment]
Genres have been added in 2.8.0b14

**Comment by OzGav** [comment]
The feature request has been implemented successfully. As such this feature request will be closed and locked. If there are enhancements desired then open a new [feature request](https://github.com/orgs/music-assistant/discussions/categories/feature-requests-and-ideas). If there is a question about how this works then check the [documentation](https://beta.music-assistant.io/), or ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord. If there are problems then ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord or open an [issue](https://github.com/music-assistant/support/issues). Thanks for your interest in seeing this idea being brought to life.
